GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -180,5 +207,29 @@ private void downloadDictionary_LinkClicked(object sender, LinkLabelLinkClickedE
         {
             Process.Start(@"https://github.com/gitextensions/gitextensions/wiki/Spelling");
         }
+
+        private void pictureAvatarHelp_Click(object sender, EventArgs e)
+        {
+            Process.Start(@"http://en.gravatar.com/site/implement/images#default-image");
+        }
+
+        private void AvatarProvider_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            ManageGravatarOptionsDisplay();
+        }
+
+        private void ManageGravatarOptionsDisplay()
+        {
+            var showAvatarOptions = (AvatarProvider)AvatarProvider.SelectedValue == GitCommands.AvatarProvider.Gravatar;
+            lblNoImageService.Visible = showAvatarOptions;
+            _NO_TRANSLATE_NoImageService.Visible = showAvatarOptions;
+            pictureAvatarHelp.Visible = showAvatarOptions;
+        }
+    }
+
+    public class ComboBoxItem<T>
```

Comment:
I'm pretty sure this can be turned into private
```suggestion
    private class ComboBoxItem<T>
```